api/src/index.ts

```diff
@@ -10,14 +10,15 @@ nconf
       host: "scholar-reader.c5tvjmptvzlz.us-west-2.rds.amazonaws.com",
       port: 5432,
       database: "scholar-reader",
-      user: "api"
+      user: "api",
+      schema: "public"
```

Comment:
Nice! Glad this exists as a connection setting.

Reply:
Me too!!